mod/wikirate_source/spec/set/type_plus_right/source/file-import_spec.rb

```diff
@@ -2,191 +2,185 @@
   before do
     login_as 'joe_user'
     test_csv = File.open("#{Rails.root}/mod/wikirate_source/spec/set/type_plus_right/source/import_test.csv")
-
-    @source = Card.create! :type_id=>Card::SourceID,:subcards=>{'+File'=>{ :file=>test_csv,:type_id=>Card::FileID}}
-    Card::Env.params["is_metric_import_update"] = 'true'
+    @source = Card.create! type_id: Card::SourceID,
+                           subcards: {
+                             '+File'=>{ file: test_csv,type_id: Card::FileID}
```

Comment:
Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside } missing.